src/python/crash_analysis/stack_parsing/stack_analyzer.py

```diff
@@ -208,7 +208,9 @@
 UBSAN_INTEGER_OVERFLOW_REGEX = re.compile(
     r'.*(integer overflow|negation of.*cannot be represented in type).*')
 UBSAN_INVALID_BOOL_VALUE_REGEX = re.compile(
-    r'.*not a valid value for type \'bool\'.*')
+    r'.*not a valid value for type \'(bool|BOOL)\'.*')
+UBSAN_INVALID_BUILTIN_REGEX = re.compile(r'.*, which is not a valid argument.*')
+UBSAN_INVALID_ENUM_VALUE_REGEX = re.compile(r'.*not a valid value for type.*')
```

Comment:
This will conflict with UBSAN_INVALID_BOOL_VALUE_REGEX
add 
" \'(enum"

Reply:
I believe this will actually use the enum type name. I don't think we can predict what those will be.